rag-advanced/README.md

```diff
@@ -0,0 +1,26 @@
+# Advanced RAG IRL
+
+This repository contains the code for the Advanced RAG IRL course.
```

Comment:
could you add a link to the course? https://www.wandb.courses/courses/rag-in-production

Reply:
before launching the course we should also add a quick overview, course learning objectives etc. but we can do so afterwards